core/controllers/learner_group.py

```diff
@@ -790,6 +860,67 @@ def put(self, learner_group_id):
         })
 
 
+class ExitLearnerGroupHandler(base.BaseHandler):
+    """Handles a learner exiting from a learner group."""
+
+    GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+    URL_PATH_ARGS_SCHEMAS = {
+        'learner_group_id': {
+            'schema': {
+                'type': 'basestring',
+                'validators': [{
+                    'id': 'is_regex_matched',
+                    'regex_pattern': constants.LEARNER_GROUP_ID_REGEX
+                }]
+            },
+            'default_value': None
+        }
+    }
+
+    HANDLER_ARGS_SCHEMAS = {
+        'PUT': {
+            'learner_username': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': constants.MAX_USERNAME_LENGTH
+                    }]
+                }
+            }
+        }
+    }
+
+    @acl_decorators.can_access_learner_groups
+    def put(self, learner_group_id):
+        """Handles PUT requests."""
+
+        learner_username = self.normalized_payload.get('learner_username')
+
+        learner_user_id = user_services.get_user_id_from_username(
+            learner_username)
+        learner_group_services.remove_learners_from_learner_group(
+            learner_group_id, [learner_user_id], True)
+
+        learner_group = learner_group_fetchers.get_learner_group_by_id(
+            learner_group_id)
```

Comment:
Again learner_group_id can be None!

Reply:
None handle not needed as group id would never be none